public/locale/en.json

```
@@ -688,6 +688,7 @@
   "click_manage_sub_locations_mobile": "Click <PenLine/>to edit and <ArrowIcon/> to manage sub-locations.",
   "click_on": "Click on",
   "click_on_map_to_select_location": "Click on the map to select the location",
+  "client_error": "Client error",
```

```suggestion
  "client_error": "Client Error",
```

Should we ? just personal choice I think 😅 , feel free to ignore it 😅

---

😂

---

@Jacobjeevan should i ??